metis_client/dtos/datasource.py

```diff
@@ -29,9 +29,6 @@ class MetisDataSourceDTO(MetisTimestampsDTO):
     type: int
     collections: Sequence[MetisCollectionDTO]
 
-    def __repr__(self):
```

Comment:
@knopki what is the proper way to do the customized printing?

Reply:
`TypedDict` can't be extended and can't be mixed with no-`TypedDict` classes. So, 

```python
def format_ds_dto(dto: MetisDataSourceDTO) -> str:
    return f'[{dto["id"]}] {dto["name"]} type {dto["type"]}'
```

🤣